src/main/resources/webapp/webapp.js

```diff
@@ -0,0 +1,187 @@
+const portalLib = require("/lib/xp/portal");
+const authLib = require("/lib/xp/auth");
+const mustacheLib = require("/lib/mustache");
+const storeLib = require("/lib/office-league-store");
+const invitationLib = require("/lib/invitation");
+const pushLib = require("/lib/push");
+const geoipLib = require("/lib/geoip");
+const router = require("/lib/router")();
+const sw = require("/lib/pwa/sw-controller.js");
+const manifest = require("/lib/pwa/manifest-controller.js");
+
+const controllers = {
+    assets: require("./controllers/assets.js"),
+    grapql: require("../services/graphql/graphql.js"),
+    playerImage: require("./controllers/player-image.js"),
+    leagueImage: require("./controllers/league-image.js"),
+    teamImage: require("./controllers/team-image.js"),
+};
+
+const view = resolve("webapp.html");
+
+// All request to the webapp
+exports.all = function (req) {
+    return router.dispatch(req);
+};
+
+// Assets where served by this endpoint
+// https://github.com/enonic/app-office-league/issues/459
+router.get("/assets/{path}/{file}", function (req) {
+    return controllers.assets.get(req);
+});
+
+router.get("/assets/img/flags/{file}", function (req) {
+    return controllers.assets.get(req);
+})
+
+// Image asset routs
+router.get("/teams/image/{id}/{name}/", function(req) {
+    return controllers.teamImage.get(req);
+});
+router.get("/players/image/{id}/{name}", function(req) {
+    return controllers.playerImage.get(req);
+});
+router.get("/leagues/image/{id}/{path}", function (req) {
+    return controllers.leagueImage.get(req);
+});
+router.get("/service-worker.js", function(req) {
+    return sw.get(req);
+});
+
+router.post("/api/graphql", function (req) {
+    return controllers.grapql.post(req);
+});
+
+router.get("/manifest.json", function(req) {
+    return manifest.get(req);
+});
+
+router.get("/", defaultRout);
+router.get("/{path}", defaultRout);
+
+function defaultRout(req) {
```

Comment:
Is this a typo? I would expect this to be `defaultRoute` with an `e` at the end 🤓